### ISSUE TITLE
Couldn't fetch audio server info: couldn't parse server version, regexp didn't match.

### ISSUE THREAD

**Issue by druskus20** [lawl/NoiseTorch#138]
I installed `noisetorch-bin 0.11.2-1` from the AUR. When I run: 
```
noisetorch -i
# or
noisetorch -u
```
I get the following error: 
```
Couldn't fetch audio server info: couldn't parse server version, regexp didn't match.
```
It doesnt matter if noisetorch is already loaded or no. 
However, using the gui tool it works flawlessly.

**Comment by BluCobalt** [comment]
Same on gentoo.

**Comment by lawl** [comment]
Can you attach a log? See [here](https://github.com/lawl/NoiseTorch/wiki/Troubleshooting).

**Comment by druskus20** [comment]
Running `noisetorch -u -log`I got: 
```Couldn't fetch audio server info: couldn't parse server version, regexp didn't match.
Error unloading PulseAudio Module: PulseAudio client was closed
```
and the log: `nt_unload.log`:
[nt_unload.log](https://github.com/lawl/NoiseTorch/files/6535353/nt_unload.log)

Running `noisetorch -i -log` I got:
```
Couldn't fetch audio server info: couldn't parse server version, regexp didn't match.
```
and the log: `nt_load.log`.
[nt_load.log](https://github.com/lawl/NoiseTorch/files/6535352/nt_load.log)

**Comment by lawl** [comment]
Thanks, I think I see what's going wrong, according to the log file though, it should still have loaded/unloaded it correctly. Can you confirm?

So this shouldn't actually impact functionality.

**Comment by druskus20** [comment]
Yes, it seems like noisetorch is being loaded/unloaded propperly

**Comment by BluCobalt** [comment]
[nt.log](https://github.com/lawl/NoiseTorch/files/6541027/nt.log)

**Comment by druskus20** [comment]
I just noticed, functionality doesnt seem affected, however, in the gui tool I get an orange warning saying `Inconsistent state, please unload first.`

**Comment by lawl** [comment]
@druskus20 this may be because you have loaded different settings via the CLI than you configured via the GUI. E.g. if you have input and output filtering checked in the GUI, noisetorch will check that both are loaded. But if you only load it for input via the CLI then the GUI will report it as inconsistent.

I think that's what may be happening here. Unfortunately that's just an artifact of mixing CLI/GUI and not possible to fix without changing how the CLI works/tying it to the UI config.

**Comment by gbrennon** [comment]
same here

**Comment by lawl** [comment]
This is fixed in [0.11.3](https://github.com/lawl/NoiseTorch/releases/tag/0.11.3).
